its/ruling/src/test/resources/expected/kotlin/kotlin-language-server/kotlin-S6519.json

```diff
@@ -0,0 +1,5 @@
+{
+'kotlin-kotlin-language-server-project:server/src/main/kotlin/org/javacs/kt/codeaction/quickfix/QuickFix.kt':[
+19,
```

Comment:
Surprised to see just one issue. I thought `ktor` should have been triggered too

Reply:
Nevermind, In Ktor we don't have kotlin stdlib binaries for now, need to change it to get proper results